### DISCUSSION TITLE
Greenlight Stream not starting

### DISCUSSION THREAD

**Discussion by goofyCS811** [unknownskl/xbox-xcloud-client#1265] [General]
I'm using Greenlight on a Macbook Pro running on macOS Ventura 13.5.2. 

Every time I try to start remote play I get this error:
<img width="1440" alt="Screenshot 2024-07-09 at 14 33 41" src="https://github.com/unknownskl/greenlight/assets/175128513/08bdc787-03c3-46e5-915d-7888412e3955">

I have tried restarting the Xbox itself, the macbook pro and the app as well but nothing seems to fix this issue. 

If anyone has any solutions to this issue they would be greatly appreciated! I just wanna play some Elden Ring 😩

**Comment by rackerby** [comment]
Assuming you have access to the console, what result do you get if you go to Settings, Network, Test remote play?

**Reply by goofyCS811** [reply]
I can't check just now as the wee man is currently using the TV to play PS but I'll let you know as soon as I get a chance. 

I did just try remote playing through my phone and it couldn't connect to the console either so I'm assuming it's an issue with my Xbox or the local Xbox services.

**Reply by goofyCS811** [reply]
I just managed to get on the Xbox and run the test and this was the result: 
<img width="487" alt="Screenshot 2024-07-09 at 15 23 12" src="https://github.com/unknownskl/greenlight/assets/175128513/095b719a-6ee1-46dc-a987-f5ab880f4b21">

**Comment by DJGM1974** [comment]
I'm also getting the same error using Greenlight on my Mac mini.  Even after restarting both the Xbox and the Mac the error still
keeps appearing. I like to stream from Xbox on my Mac with something else on my main TV, but cannot do so with this error.
